src/ISO3166DataValidator.php

```diff
@@ -28,8 +26,6 @@ public function validate(array $data)
     }
 
     /**
-     * @param array $entry
```

Comment:
Whats the use case to remove this where the docs are valid?

Reply:
I'm actually ok with removing docs that cover things that are already covered by type hints now a days